src/index.css

```diff
@@ -1,13 +1,339 @@
-body {
+body, html {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
     'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  overflow-y: hidden;
+  overflow-x:auto;
+  /*max-width: 100%;*/
+  padding: 0;
 }
 
+
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+.input {
+  padding: 16px 0;
+  background: white;
+  width: 100%;
+}
+
+.inputButtons {
+  display: flex;
+  justify-content: space-between;
+}
+
+.attachIcon{
+  background-image: url("icons/attachIcon/attachIcon.svg");
+  background-size: cover;
+  padding: 1px;
+  width: 20px;
+  height: 20px;
+}
+
+.file-input-container {
+  position: relative;
+  display: inline-block;
+  cursor: pointer;
+}
+/* Hide the original file input */
+.file-input-container input[type="file"] {
+  position: absolute;
+  left: 0;
+  top: 0;
+  opacity: 0;
+}
+
+.attachIcon:hover{
+  background-color: rgba(32, 31, 30, 0.04);
+}
+
+/*.chatIcon {*/
+/*  height: 30px;*/
+/*}*/
+
+.inputMessage {
+  border: 1px solid rgba(161, 159, 157, 1);
+  padding: 8px 16px;
+  align-items: center;
+  border-radius: 5px;
+  margin-bottom: 5px;
+  max-height: 50px;
+  overflow-y: auto;
+}
+
+.messageList {
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+  height: 100vh;
+  padding: 0 10px;
+  width: calc(100% - 370px)
+}
+
+
+[contenteditable]:focus {
+  outline: 0 solid transparent;
+}
+
+[placeholder]:empty:before {
+  content: attr(placeholder);
+  color: rgba(161, 159, 157, 1)
+}
+
+[placeholder]:empty:focus:before {
+  content: "";
+}
+
+.header {
+  display: flex;
+  flex-grow: 1;
+  gap: 10px;
+  font-weight: bold;
+  padding: 5px 10px;
+  border-bottom: 1px solid rgba(32, 31, 30, 0.12);
+}
+
+.newMessage {
+  width: 100%;
+  text-align: center;
+  background: rgba(64, 126, 201, 0.08);
+  font-size: 12px;
+  color: rgba(64, 126, 201, 1);
+  margin-bottom: 10px;
+  padding: 3px 0;
+}
+
+
+
+.avatar {
+  border-radius: 4px;
+}
+
+.message {
+  position: relative;
+  display: flex;
+  gap: 8px;
+  width: 50%;
+  padding: 15px;
+}
+
+.messageText {
+  position: relative;
+  /*display: flex;*/
+  padding: 8px 12px;
+  background: rgba(243, 243, 243, 1);
+  border-radius: 4px;
+}
+
+.myMessage {
+  position: relative;
+  display: flex;
+  float: right;
+  margin-left: auto;
+  padding: 15px;
+  width: 50%;
+}
+
+.myMessageText {
+  position: relative;
+  /*display: flex;*/
```

Comment:
Please remove comments which are useless

Reply:
done, also position:relative is removed